<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>game</string>
  </property>
  <widget class="QTextBrowser" name="str">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>10</y>
     <width>841</width>
     <height>281</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="wu">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>299</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入区域：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="usestr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>1011</width>
     <height>281</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditorInteraction</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="seconds">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="begin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="daTa">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
